static/js/src/advantage/subscribe/renderers/form-renderer.js

```diff
@@ -188,7 +188,13 @@ function renderSummary(state) {
       trialUnavailableSection.classList.add("u-hide");
     } else {
       freeTrialLabel.classList.add("u-hide");
-      trialUnavailableSection.classList.remove("u-hide");
+
+      // Remove this once we want to release free trial
```

Comment:
I wonder if it would be helpful to introduce something resembling a feature flag, would remove the need for a comment and make it easier to update.

`const getIsFreeTrialEnabled () => location.search.includes("test_backend=true")`

Reply:
Good idea!